setup.py

```diff
@@ -0,0 +1,23 @@
+from setuptools import setup
+
+setup(
```

Comment:
I'm totally new to packaging python projects, too. It looks like a [setup.cfg](https://setuptools.pypa.io/en/latest/userguide/declarative_config.html) is the _new_ way to package stuff up? 🤷

Reply:
I was separately curious about the current best practices, and found this: https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html
Today I intend to read both.

Reply:
Definitely behind on the times, will read through that and try to use the latest practices!